care/emr/utils/expression_evaluator.py

```
@@ -0,0 +1,12 @@
+from evalidate import EvalException, Expr, base_eval_model
+
+expression_model = base_eval_model.clone()
+expression_model.nodes.append("JoinedStr")
+expression_model.nodes.append("FormattedValue")
+
+
+def evaluate_expression(expression: str, context: dict):
+    try:
+        return Expr(expression, model=expression_model).eval(context)
+    except EvalException as e:
+        raise ValueError(e)
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)